### DISCUSSION TITLE
Metrics dictionary needs to be complete?

### DISCUSSION THREAD

**Discussion by Marcovnr** [winedarksea/AutoTS#154] [Q&A]
Hi, this is going to be a quick question but I was not able to find out a answer by myself. 

When you pass the metrics dictionary with only one metri, for example {mae : 10}. Does it set all other metrics to zero or they remain with their default values?

**Comment by winedarksea** [comment]
They are filled with zero. So if they are missing, they are not used.
Actual code is near the bottom of the auto_model.py file. It is .get("mae_weighting", 0) etc.
